### PR TITLE
Added hideView and showView methods for child views

### PR DESCRIPTION
**Pull request by msya**
Add these two new methods:

```
ViewUtils.showView(parentView, R.id.childViewId);
ViewUtils.hideView(parentView, R.id.childViewId);
```

### CODE REVIEW

File: caffeine/src/main/java/com/percolate/caffeine/ViewUtils.java [hunk 1]
```diff
@@ -14,14 +14,17 @@
  * <h2>Utility methods that make working with views easier, less error prone, and more concise.</h2>
  * <p/>
  * <h3>Common uses:</h3>
+ * <code>LinearLayout linearLayout = ViewUtils.{@link #findViewById findViewById}(this, R.id.my_linear_layout);</code> //<b>no more casting!</b><br />
```
Comment (brentwatson) [inline caffeine/src/main/java/com/percolate/caffeine/ViewUtils.java]: Move below `findViewById` with `context` examples.  Those are more common and this doc pops up in the IDE.

File: caffeine/src/main/java/com/percolate/caffeine/ViewUtils.java [hunk 2]
```diff
@@ -238,19 +241,49 @@ public static void hideView(Activity context, int id) {
     }
 
     /**
+     * Sets visibility of the given view to <code>View.GONE</code>.
+     *
+     * @param parentView The View used to call findViewId() on.
+     * @param id         R.id.xxxx value for the view to hide "expected textView to throw a ClassCastException" + textView.
+     */
+    public static void hideView(View parentView, int id) {
+          View view = parentView.findViewById(id);
```
Comment (brentwatson) [inline caffeine/src/main/java/com/percolate/caffeine/ViewUtils.java]: `context` versions check if `context != null`,  Let's do the same in these 2 methods with `parentView`.

File: tests/src/androidTest/java/com/percolate/caffeine/testapp/ViewUtilsTest.java [hunk 3]
```diff
@@ -125,6 +125,21 @@ public void run() {
         assertEquals(View.GONE, textView.getVisibility());
     }
 
+    public void testHideViewChildView(){
+        final LinearLayout layout = ViewUtils.findViewById(getActivity(), R.id.layout_1);
```
Comment (brentwatson) [inline tests/src/androidTest/java/com/percolate/caffeine/testapp/ViewUtilsTest.java]: Do these tests pass?  I don't see `layout_1` added to the Testing App.

Comment (msya) [inline tests/src/androidTest/java/com/percolate/caffeine/testapp/ViewUtilsTest.java]: Yes. The layout `activity_view_utils ` contains a LinearLayout with id `layout_1`.

**Pull request merged**